diff --git a/node_modules/snarkjs/build/main.cjs b/node_modules/snarkjs/build/main.cjs
index 5a91ec6..7ada999 100644
--- a/node_modules/snarkjs/build/main.cjs
+++ b/node_modules/snarkjs/build/main.cjs
@@ -5499,7 +5499,7 @@ async function phase2verifyFromInit(initFileName, pTauFileName, zkeyFileName, lo
 
     if (logger) logger.info("ZKey Ok!");
 
-    return true;
+    return mpcParams;
 
 
     async function sectionHasSameRatio(groupName, fd1, sections1, fd2, sections2, idSection, g2sp, g2spx, sectionName) {
